user_interface/src/components/ModeSelector.vue

```diff
@@ -20,8 +20,11 @@
           <b-button v-if="!isSelectModeDrawBox" @click="onReset" variant="outline-info"
             >Reset</b-button
           >
-          <b-button v-if="!isSelectModeDrawBox" @click="toggleShowNumbers" variant="outline-info"
-            >{{ this.getShowNumberMode ? 'Hide' : 'Unhide' }} Numbers</b-button
+          <b-button v-if="!isSelectModeDrawBox" @click="toggleShowOrder" variant="outline-info"
+            >{{ this.showOrderMode ? 'Hide' : 'Unhide' }} Order</b-button
+          >
+          <b-button v-if="!isSelectModeDrawBox" @click="toggleHandStart" variant="outline-info"
+            >{{ this.handStartMode }}-HandStart</b-button
```

Comment:
This should actually be HoldStart instead of hand start.

Reply:
Now only got handstart. Holdstart may not may not be confusing LOL (Imagine all the markings). But yeah I get that its more similar to comp style? Can do nxt PR